openghg/dataobjects/_obsdata.py

```diff
@@ -84,46 +84,18 @@ def plot_timeseries(
         xlabel: Optional[str] = None,
         ylabel: Optional[str] = None,
         units: Optional[str] = None,
+        logo: Optional[bool] = True,
     ) -> go.Figure:
         """Plot a timeseries"""
-
-        species = self.metadata["species"]
-        site = self.metadata["site"]
-        inlet = self.metadata["inlet"]
-
-        if title is None:
-            title = f"{species.upper()} at {site.upper()} - {inlet}"
-
-        if xlabel is None:
-            xlabel = "Date"
-
-        if ylabel is None:
-            ylabel = "Concentration"
-
-        if units is not None:
-            ylabel += f"  ({units})"
-
-        data = self.data
-        x_data = data.time
-
-        try:
-            y_data = data[species]
-        except KeyError:
-            y_data = data["mf"]
-
-        font = {"size": 14}
-
-        title_layout = {"text": title, "y": 0.9, "x": 0.5, "xanchor": "center", "yanchor": "top"}
-
-        layout = go.Layout(
-            title=title_layout,
-            xaxis=dict(title=xlabel),
-            yaxis=dict(title=ylabel),
-            font=font,
+        import openghg.plotting
```

Comment:
This feels pedantic but we usually do `from openghg.plotting import plot_timeseries`, be good to keep our import style consistent.

Reply:
Yeah that's fine - my only concern was namespacing as then we'd have two `plot_timeseries` functions and wasn't sure if that would potentially cause a problem? But happy to change it to be consistent if that isn't the case.

Reply:
Ok so I've updated it to:
```from openghg.plotting import plot_timeseries as general_plot_timeseries```
to make sure it's clear where this is coming from.

Is that consistent enough with the previous style?

Reply:
Yeah looks good